tests/core/full_node/test_mempool.py

```diff
@@ -80,19 +79,6 @@ def generate_test_spend_bundle(
     return transaction
 
 
-# this is here to avoid this error:
-# ScopeMismatch: You tried to access the 'function' scoped fixture 'event_loop'
-#   with a 'module' scoped request object, involved factories
-@pytest.fixture(scope="module")
-def event_loop():
-    loop = asyncio.get_event_loop()
-    yield loop
-
-
-# TODO: this fixture should really be at function scope, to make all tests independent.
-# The reason it isn't is that our simulators can't be destroyed correctly, which
-# means you can't instantiate more than one per process, so this is a hack until
-# that is fixed. For now, our tests are not independent
```

Comment:
I believe this comment is still relevant

Reply:
https://gist.github.com/altendky/8128bf09b153034a38edf09f60a111c1 It looks like this now runs ok with the fixture set to function scope.  I left it at module scope for later consideration of performance tradeoffs.